src/test/java/com/e451/service/TransactionServiceTest.java

```diff
@@ -0,0 +1,186 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by e978726 on 2/28/2017.
+ */
+public class TransactionServiceTest {
+    // Constants for testing
+    private final Long ID = new Long(1);
+    private final Long HOUSEHOLD_KEY = new Long(1);
+    private final Long BASKET_ID = new Long(2);
+    private final Long DAY = new Long(8);
+    private final Long PRODUCT_ID = new Long(3);
+    private final Long QUANTITY = new Long(4);
+    private final Double SALES_VALUE = 1.25;
+    private final Long STORE_ID = new Long(5);
+    private final Double COUPON_MATCH_DISCOUNT = .55;
+    private final Double COUPON_DISC = .10;
+    private final Double RETAIL_DISC = .05;
+    private final Long TRANS_TIME = new Long(1425);
+    private final Long WEEK_NO = new Long(6);
+
+    private final Long ID2 = new Long(2);
+    private final Long HOUSEHOLD_KEY2 = new Long(2);
+    private final Long BASKET_ID2 = new Long(3);
+    private final Long DAY2 = new Long(8);
+    private final Long PRODUCT_ID2 = new Long(4);
+    private final Long QUANTITY2 = new Long(5);
+    private final Double SALES_VALUE2 = 5.25;
+    private final Long STORE_ID2 = new Long(6);
+    private final Double COUPON_MATCH_DISCOUNT2 = .25;
+    private final Double COUPON_DISC2 = .15;
+    private final Double RETAIL_DISC2 = .07;
+    private final Long TRANS_TIME2 = new Long(1725);
+    private final Long WEEK_NO2 = new Long(7);
+
+    private TransactionService testTransactionService;
+    private Pageable mockPageable = mock(Pageable.class);
+    private TransactionRepository TransactionRepository = mock(TransactionRepository.class);
+    private List<TransactionRecord> TransactionRecords = new ArrayList<>(2);
+
+    @Before
+    public void setUp() {
+        testTransactionService = new TransactionServiceImpl(TransactionRepository);
+
+    }
+
+    @Test
+    public void testTransactionServiceShouldConstruct() {
+        Assert.assertNotNull("Transaction service was not instantiated"
+                , testTransactionService);
+    }
+
+    @Test
+    public void testTransactionServiceShouldReturnTransactions() {
+        TransactionRecords.add(new TransactionRecord(
+                ID,
+                HOUSEHOLD_KEY,
+                BASKET_ID,
+                DAY,
+                PRODUCT_ID,
+                QUANTITY,
+                SALES_VALUE,
+                STORE_ID,
+                COUPON_MATCH_DISCOUNT,
+                COUPON_DISC,
+                RETAIL_DISC,
+                TRANS_TIME,
+                WEEK_NO
+        ));
```

Comment:
Please make your parens and whitespace usage consistent on method calls. In some places, you have your closing parens on the same line as the final parameter. In others, you have it on a separate line. (Incidentally, I'd prefer them on the same line as the final parameter, but as long as it's consistent across the file, I'm ambivalent.)